easybuild/easyconfigs/g/graph-tool/graph-tool-2.45-foss-2022a.eb

```diff
@@ -0,0 +1,63 @@
+# This version does require cairomm 1.14 as there has been a change in the API!
+# See https://forum.skewed.de/t/error-fill-rule-even-odd-is-not-a-member-of-cairo/1397
+# It should be fixed in the upcoming version though. 
```

Comment:
@sassy-crick Can you check whether a more recent version of `graph-tool` is now available which is compatible with `cairomm` 1.16.x, so we don't need to stick to the older 1.14.x?